transport/src/main/java/io/netty/bootstrap/FailedChannel.java

```diff
@@ -40,7 +40,7 @@ protected AbstractUnsafe newUnsafe() {
 
     @Override
     protected boolean isCompatible(EventLoop loop) {
-        return false;
+        return true;
```

Comment:
on purpose?

Reply:
Yes as its called from within the constructor. That said the code is removed in later prs that depend on this one.